src/main/java/seedu/address/model/person/ClassTiming.java

```diff
@@ -33,6 +39,9 @@ public ClassTiming(String classTiming) {
         requireNonNull(classTiming);
         checkArgument(isValidClassTiming(classTiming), MESSAGE_CONSTRAINTS);
         value = formatClassTiming(classTiming);
+        startTime = parseStartTime(value);
+        endTime = parseEndTime(value);
+        day = parseDay(value);
```

Comment:
Should this be an enum instead?

Reply:
I think day we can use enum but not for start and end time